build-repo.sh

```diff
@@ -51,10 +55,14 @@ echo
 if [ ! -f ${PACKAGER_PRIVKEY} ]; then
   echo "======== WARN: PACKAGER_PRIVKEY is not present ======="
   abuild-keygen -a -i -n
-  sudo cp ${HOME}/.abuild/*.pub /etc/apk/keys/
   RESIGN=true
+else
+  echo "Using ${PACKAGER_PRIVKEY}"
+  openssl rsa -in ${PACKAGER_PRIVKEY} -pubout | sudo tee ${PACKAGER_PRIVKEY}.pub
 fi
```

Comment:
Out of curiosity, why was this not necessary before?

Reply:
I guess it depends on the abuild version.